packages/odie-client/src/data/use-send-odie-message.ts

```diff
@@ -56,7 +56,11 @@ export const useSendOdieMessage = () => {
 		If the message is a request for human support, it will escalate the chat to human support, if eligible.
 		If email support is forced, it will add an email fallback message.
 	*/
-	const addMessage = ( message: Message | Message[], props?: Partial< Chat > ) => {
+	const addMessage = (
+		message: Message | Message[],
+		props?: Partial< Chat >,
+		fromError: boolean = false
+	) => {
```

Comment:
Having a single object as the function arguments lets you avoid chaining optional empty params on the calls, which are easy to forget.

```suggestion
	const addMessage = ( {
		message,
		props = {},
		fromError = false,
	}: {
		message: Message | Message[];
		props?: Partial< Chat >;
		fromError: boolean;
	} ) => {
```